env/production/cloudwatch_alarms/terragrunt.hcl

```diff
@@ -4,6 +4,7 @@ inputs = {
   sns_topic_critical_name  = "alert-critical"
 
   feature_api_alarms              = true
+  api_gateway_400_error_threshold = 100
```

Comment:
These alarms are going to go off all the time, we need to set this much much higher in production.

Probably higher then our highest spike over the last month.

Reply:
Good catch - I've set it to 3000 which is double our highest spike (1,470).